### PR TITLE
Exposing the icann status of the TLD

### PR DESCRIPTION
**Pull request by jmdacruz**
Using the `PublicSuffix` function in order to tell if the TLD is managed by ICANN, and adding this info to the URL struct

### CODE REVIEW

File: parse.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ import (
 //URL embeds net/url and adds extra fields ontop
 type URL struct {
 	Subdomain, Domain, TLD, Port string
+	Icann                        bool
```
Comment (jpillora) [inline parse.go:17]: should be `ICANN` Go doesn't title-ise acronyms

Comment (jpillora) [reply parse.go:17]: we could also make it `Managed bool`?  or maybe would `ICANN` be less confusing?

**Comment by jpillora** [comment]
Thanks for the PR :)

File: parse.go (hunk 1 above)
Comment (jmdacruz) [reply parse.go:17]: I think that ICANN is probably less confusing, will make the change if that's ok

**Comment by jpillora** [comment]
merged :)

**Comment by jmdacruz** [comment]
awesome, thanks @jpillora! If you can, please add a release/tag so that I can reference this version directly.

**Comment by jpillora** [comment]
tagged as `v1.1.0`

**Pull request merged**